template.rb

```diff
@@ -0,0 +1,83 @@
+# Include govuk-rails-app-template root in source_paths
+source_paths << File.dirname(__FILE__)
+
+run 'bundle install'
+git :init
+git add: "."
+git commit: "-a -m 'Bare Rails application'"
+
+# Configure JSON-formatted logging
+gem 'logstasher'
+run 'bundle install'
+# Enable JSON-formatted logging in production
+environment nil, env: "production" do <<-'RUBY'
+config.logstasher.enabled = true
+  config.logstasher.logger = Logger.new(Rails.root.join("/log/production.json.log"))
+  config.logstasher.suppress_app_log = true
+RUBY
+end
+# Remove the default log formatter
+gsub_file 'config/environments/production.rb', 'config.log_formatter = ::Logger::Formatter.new', '# config.log_formatter = ::Logger::Formatter.new'
+
+# Configure JSON-formatted logging with additional fields
+initializer "logstasher.rb", File.read("#{File.dirname(__FILE__)}/templates/initializers/logstasher.rb")
+
+git add: "."
+git commit: "-a -m 'Use logstasher for JSON-formatted logging in production'"
+
+# Setup rspec
+gem_group :development, :test do
+  gem 'rspec-rails'
```

Comment:
It would be good to pin the version here to `'~> 3.3'`.  They follow semantic versioning - http://www.relishapp.com/rspec/docs/versioningpolicy.